ext/tidy/tidy.c

```diff
@@ -720,28 +667,19 @@ static void *php_tidy_get_opt_val(const PHPTidyDoc *ptdoc, TidyOption opt, TidyO
 {
 	*type = tidyOptGetType(opt);
 
-	switch (*type) {
-		case TidyString: {
-			const char *val = tidyOptGetValue(ptdoc->doc, tidyOptGetId(opt));
-			if (val) {
-				return (void *) zend_string_init(val, strlen(val), 0);
-			} else {
-				return (void *) ZSTR_EMPTY_ALLOC();
-			}
+	if (*type == TidyString) {
```

Comment:
what s wrong with switch/case ? for only 3 possibilities (and it is not going to change anytime soon), I m not against just to see if this is just taste or anything.